package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "openwpm",
+  "version": "0.10.0",
```

Comment:
I think we should remove it as long as npm doesn't complain to hard. But also remove it from the two other package.json files (which have not been updated in releases). I put it in because the other two package.json files had a version and 0.10 seemed like as good as any to pick.

Reply:
Resolved in 7dcebfc